### PR TITLE
[IAED] Cleanup Cheatsheets

### PR DESCRIPTION
**Pull request by randomicecube**

### CODE REVIEW

File: content/iaed/1000-algoritmos.sheet.md [hunk 1]
```diff
@@ -20,45 +20,381 @@ O código disponível nesta página foi retirado de slides dos docentes, bibliog
...
+    if (less(B, A))    \
+    exch(A, B)
+```
+
+Mais ainda, de realçar que todos os algoritmos apresentados nesta secção estão explicados na secção respetiva dos resumos, indicada a par do respetivo código.
 
 ### Selection Sort
 
-`embed:assets/1000-selection-sort.c`
+[Explicação nos resumos](../algoritmos-elementares-ordenacao#selection-sort).
```
Comment (diogotcorreia) [inline content/iaed/1000-algoritmos.sheet.md:45]: Even though it works, I prefer for all links to be absolute (i.e. `/iead/algoritmos-elementares-ordenacao#selection-sort`)
Same for the other links below

File: content/iaed/1000-algoritmos.sheet.md [hunk 2]
```diff
@@ -20,45 +20,381 @@ O código disponível nesta página foi retirado de slides dos docentes, bibliog
...
+#define MAX 100 // 100 é placeholder
+/* vamos evitar o uso de variáveis na declaração de vetores,
+ * porque variable length arrays are bad
+ * devemos OU usar constantes OU usar malloc e amigos
+ * wikipedia article relevante:
+ * https://en.wikipedia.org/wiki/Variable-length_array#C99
+ */
+
+void countingSort(int numbers[], int size) {
+  int count[SIZE];
```
Comment (diogotcorreia) [inline content/iaed/1000-algoritmos.sheet.md:266]: Just use `malloc` here tbh

File: content/iaed/1000-algoritmos.sheet.md [hunk 3]
```diff
@@ -20,45 +20,381 @@ O código disponível nesta página foi retirado de slides dos docentes, bibliog
...
+
+```c
+#define maxN 100000
+#define bitsword 32
+#define bitsbyte 8
+#define bytesword 4
+#define R (1 << bitsbyte)
+#define digit(n, w) ((n & ( 1 << w )) >> w)
+
+// defines um pouquinho cursed, mas é o que há
```
Comment (diogotcorreia) [inline content/iaed/1000-algoritmos.sheet.md:301]: Vê o que tenho mesmo nos resumos, provavelmente é melhor

File: content/iaed/1000-algoritmos.sheet.md (hunk 1 above)
Comment (randomicecube) [reply content/iaed/1000-algoritmos.sheet.md:45]: Fixed in the next commit

File: content/iaed/1000-algoritmos.sheet.md (hunk 2 above)
Comment (randomicecube) [reply content/iaed/1000-algoritmos.sheet.md:266]: Fixed in the next commit

File: content/iaed/1000-algoritmos.sheet.md (hunk 3 above)
Comment (randomicecube) [reply content/iaed/1000-algoritmos.sheet.md:301]: Fixed in the next commit

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Move cheatsheets to `content/iaed/cheatsheets`

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**